### ISSUE TITLE
Modifiers?

### ISSUE THREAD

**Issue by jens1o** [coderaiser/rendy#1]
It would be awesome if you were able to manually define some modifiers, that, as it says, modifies the data before outputting.

Something like 

```
{{ users|implode:',' }}
```

Would make something like `jens1o, test, mickey` when you pass an array with
```
[
    'jens1o',
    'test',
    'mickey'
]
```

Regards!

Jens

**Comment by coderaiser** [comment]
Are you talking about custom modifiers or predefined?

**Comment by jens1o** [comment]
Both actually, so it is flexible. Otherwise, if you're open to add new modifiers in the future, that could be an option, too.

**Comment by coderaiser** [comment]
I think it can complicate `rendy` a lot, anyways this can be achived with some module which process all modifiers and then calls rendy. Do you know any modules which can do such things?

**Comment by jens1o** [comment]
No, unfortunately not. But I'd need such an module, too.

**Comment by coderaiser** [comment]
Closed due to a long time of inactivity 🏝

**Comment by jens1o** [comment]
Still a feature request I'd look forward to. :/

**Comment by coderaiser** [comment]
@jens1o what is your use case? And why don't you use something like [handlerbars](https://handlebarsjs.com/) for this purpose? It has a powerful system of helpers.
What is wrong with existing libraries with modifiers?

**Comment by jens1o** [comment]
I don't understand what you're saying. The readme says that it is compatible with `handlebars`, and now you're passing me to use the original one instead?

**Comment by coderaiser** [comment]
Could you please show me the place, where handlebars used with modifiers in the way you use it?

How would you implement modifiers support?
Maybe [nearly](https://nearley.js.org) can be used for this purpose?

**Comment by coderaiser** [comment]
Just landed support of modifiers in [v4.1.0](https://github.com/coderaiser/rendy/releases/tag/v4.1.0)🎉. It was hard but I did my best to preserve speed and simplicity of **Rendy**.

Is it works for you?
